cmd/lakectl/cmd/root.go

```diff
@@ -129,6 +129,11 @@ const (
 
 	defaultSyncParallelism = 25
 	defaultSyncPresign     = true
+
+	myRepoExample   = "lakefs://my-repo"
+	myBucketExample = "s3://my-bucket"
+	myBranchExample = "my-branch"
+	runIDExample    = "20230719152411arS0z6I"
```

Comment:
```suggestion
	myRunIDExample    = "20230719152411arS0z6I"
	myDigest          = "600dc0ffee"
```
(and use that same digest in all messages)